code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -699,77 +699,85 @@
 ////////////////
 /* Ammo Boxes */
 ////////////////
-/obj/item/storage/box/slug
-	name = "ammunition box (Slug)"
+
+// Empty type
+// All these can only hold their special ammo types and automatically are filled with them
```

Comment:
need to update these 2 comments. Empty type is also a very vague comment and probably just should be removed

Reply:
Removed!